app/js/components/manageApps/ManageApps.jsx

```diff
@@ -390,18 +390,16 @@ export default class ManageApps extends React.Component {
         });
       } else {
         searchResults.forEach(result => {
-          if (result.type == "OWA") {
-            axios.get(`https://addons.openmrs.org/api/v1//addon/${result.uid}`)
-              .then(res => {
-                this.setState((prevState, props) => {
-                  return {
-                    appList: [res.data],
-                    downloadUri: res.data['versions'][0].downloadUri,
-                    install: true
-                  };
-                });
+          axios.get(`https://addons.openmrs.org/api/v1//addon/${result.uid}`)
+            .then(res => {
+              this.setState((prevState, props) => {
+                return {
+                  appList: [res.data],
+                  downloadUri: res.data['versions'][0].downloadUri,
```

Comment:
Seems you are using 'res' to mean 'response', could you use the full word to make it clearer.

Reply:
spot on! 👍

Reply:
Noted